plugins/javadoc/src/test/kotlin/javadoc/JavadocClasslikeTemplateMapTest.kt

```diff
@@ -257,19 +257,19 @@ internal class JavadocClasslikeTemplateMapTest : AbstractJavadocTemplateMapTest(
             assertParameterNode(
                 node = first,
                 expectedName = "simple",
-                expectedType = "<a href=.html>String</a>",
```

Comment:
If you could add external documentation links to the test it would be great. Since those should be links (and are in normal javadoc). It should be as simple as adding:
```
externalDocumentationLinks = listOf(
                        externalLink("https://docs.oracle.com/javase/8/docs/api/"),
                        externalLink("https://kotlinlang.org/api/latest/jvm/stdlib/")
)
```
To configuration

Reply:
Done